<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>del</class>
 <widget class="QDialog" name="del">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;信息删除&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>591</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deleteline">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入编号：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="carradio">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>汽车表</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sellradio">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>供应商表</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="needradio">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>需求表</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
